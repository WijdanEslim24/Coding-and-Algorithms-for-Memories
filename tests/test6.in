9
qwerfghjklzxcvmwertp
wetyuipasdklzcvbnqio
qweriopsdfghzbnweuio
qwrtioadfzcvbnrtyiop
rtuasdfghjkzxcbmeuip
hjklzxcvnmqwertyuiop
wrtuioadfghjkzxrtyip
qwertyuiopaswertyuip
qwertyuioasdfghjklzx
16