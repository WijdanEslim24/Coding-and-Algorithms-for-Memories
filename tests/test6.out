SCS: qwertyuiopasdfghjklzxcvbnmqwertyuiop
SCS Length: 36